app/feeds/new/license/route.js

```diff
@@ -2,7 +2,13 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
 	createFeedFromGtfsService: Ember.inject.service('create-feed-from-gtfs'),
-	  model: function() {
-	    return this.get('createFeedFromGtfsService').feedModel;
-	  }
+
+	model: function() {
+		var feedModel = this.get('createFeedFromGtfsService').feedModel;
+		if (feedModel == null) {
+			this.transitionTo('feeds.new');
```

Comment:
The logic on lines 7-10 is what I think you should wrap up in a method on the `createFeedFromGtfsService` service. Name it something like `getFeedOrTransitionToStart` and then you can call that method from the operator route, the license route, and the submit route.